code/datums/mutations.dm

```diff
@@ -646,6 +646,60 @@ GLOBAL_LIST_EMPTY(mutations_list)
 	if(owner.a_intent == INTENT_HARM)
 		owner.LaserEyes(target)
 
+/datum/mutation/human/healing_hands
```

Comment:
Please make this clear that this will automatically overwrite an existing mutation if the DNA block count is not raised.

Reply:
ok

Reply:
gimme a minute